src/app/app.component.ts

```diff
@@ -1,16 +1,17 @@
 import { Component, OnInit } from '@angular/core';
```

Comment:
Could you not change the existing indentation for this file? It goes as a file change in git.

Reply:
let us do this right. we can keep the code readable. its something to fix the indentation.